source/reference/minio-mc/mc-license-info.rst

```diff
@@ -0,0 +1,71 @@
+===================
+``mc license info``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc license info
+
+Description
+-----------
+
+The :mc-cmd:`mc license info` command displays information about the cluster's license.
```

Comment:
about the MinIO deployment's license status. Specifically, whether the deployment uses the AGPLv3 Open Source license *or* the MinIO Commercial License

Reply:
👍